eng/ignore-links.txt

```diff
@@ -1 +1,2 @@
-# Placeholder file for links that need exceptions from link checking
\ No newline at end of file
+# Placeholder file for links that need exceptions from link checking
+https://github.com/Azure/azure-sdk-for-ios/releases/tag/1.0.0-beta.2
```

Comment:
I'd like us to try and avoid having links in the repo that aren't valid even if they aren't valid even temporarily. We should instead try and stick with links that exist already.